### PR TITLE
[jvm-packages] Expose prediction feature contribution on the Java side 

### PR DESCRIPTION
**Pull request by edi-bice**
Added a method similar to the predictLeaf one, called predictContrib, and followed that pattern.

### CODE REVIEW

**Comment by CodingCat** [comment]
please add [jvm-packages] in the title when you are modifying jvm-packages,

**Comment by CodingCat** [comment]
would you please also add scala version for this?

**Comment by edi-bice** [comment]
Sure thing. I wasn't even aware of it.

File: jvm-packages/xgboost4j/src/main/scala/ml/dmlc/xgboost4j/scala/Booster.scala [hunk 1]
```diff
@@ -141,6 +141,20 @@ class Booster private[xgboost4j](private var booster: JBooster)
...
+    * Output feature contributions toward predictions of given data
+    *
+    * @param data      dmatrix storing the input
+    * @param treeLimit Limit number of trees in the prediction; defaults to 0 (use all trees).
+    * @return The feature contributions and bias.
+    * @throws XGBoostError native error
+    */
+  @throws(classOf[XGBoostError])
+  def predictContrib(data: DMatrix, treeLimit: Int = 0)
+  : Array[Array[Float]] = {
```
Comment (CodingCat) [inline jvm-packages/xgboost4j/src/main/scala/ml/dmlc/xgboost4j/scala/Booster.scala]: nit: move this line to the above one (if it does not exceed 100 chars)

**Comment by CodingCat** [comment]
LGTM, just a nit comment

**Comment by CodingCat** [comment]
merged , thanks

**Comment by edi-bice** [comment]
thanks @CodingCat

**Pull request merged**